test/controllers/questions_controller_test.rb

```diff
@@ -28,27 +36,23 @@ class AuthenticatedTest < ActionDispatch::IntegrationTest
 
       get api_questions_url, headers: { 'Authorization' => api_key }
 
-      assert_response :success
-      assert expected_response == response.body, "\nExpected:\n#{expected_response} \n\nActual:\n#{response.body.to_json}"
+      expect_successful_response(expected_response)
     end
 
     test 'counts api_key utilization' do
       tenant = create :tenant
-      Redis.current.set tenant.id, 0
-
-      assert 0 == tenant.usage_count, "Wrong tenant usage count.\nExpected: 0\nActual: #{tenant.usage_count}"
+      tenant_requests = 4
 
-      get api_questions_url, headers: { 'Authorization' => tenant.api_key }
-      get api_questions_url, headers: { 'Authorization' => tenant.api_key }
-      get api_questions_url, headers: { 'Authorization' => tenant.api_key }
-      get api_questions_url, headers: { 'Authorization' => tenant.api_key }
+      tenant_requests.times.each do
```

Comment:
I thought this was a bit more clear - but I may be wrong. I didn't like the 4 below being a magic number.

Reply:
Let's leave magic to ~assembly devs~ real magicians. 😄